ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-ec.yaml

```diff
@@ -1579,7 +1579,7 @@ tests:
     workflow: baremetal-lab-agent-install-pxe
 - as: baremetal-pxe-ha-agent-ipv4-static-connected-f14
   cluster: build05
-  cron: 30 14 5,19 * *
+  cron: 30 14 11,26 * *
```

Comment:
This one and the above one are running at the same date/time.

Reply:
@liangxia i had observed other baremetal jobs, and looks 11,26 is having less jobs. so used it twice for ec jobs.